pkg/controller/iam/policymember/iampolicymember_controller.go

```diff
@@ -164,6 +165,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		}
 		return reconcile.Result{}, err
 	}
+	// r.Get() overrides the TypeMeta to empty value, so need to configure it
+	// after r.Get().
+	memberPolicy.TypeMeta = metav1.TypeMeta{
```

Comment:
Can we use `memberPolicy.SetGroupVersionKind(iamv1beta1.IAMPolicyMemberGVK)` -- I think it's equivalent?

Reply:
Good point. Done!